misk/src/test/kotlin/misk/web/interceptors/ProtobufInterceptorFactoryTest.kt

```diff
@@ -18,50 +17,45 @@ import okio.Pipe
 import org.junit.Rule
 import org.junit.Test
 import javax.inject.Inject
-import kotlin.reflect.KFunction
 
 class ProtobufInterceptorFactoryTest {
-    @get:Rule
-    val miskTestRule = MiskTestRule()
+  @get:Rule
+  val miskTestRule = MiskTestRule()
 
-    @Inject internal lateinit var protobufAction: ProtobufAction
-    @Inject internal lateinit var protobufInterceptorFactory: ProtobufInterceptorFactory<*, *>
-    @Inject internal lateinit var boxResponseInterceptorFactory: BoxResponseInterceptorFactory
+  @Inject internal lateinit var protobufAction: ProtobufAction
+  @Inject internal lateinit var protobufInterceptorFactory: ProtobufInterceptorFactory<*, *>
+  @Inject internal lateinit var boxResponseInterceptorFactory: BoxResponseInterceptorFactory
 
-    @Test
-    fun test() {
-        val action = protobufAction::hello.asAction()
+  @Test
+  fun test() {
+    val action = ProtobufAction::hello.asAction()
 
-        val boxResponseInterceptor: Interceptor = boxResponseInterceptorFactory.create(action)!!
-        val protobufInterceptor: ProtobufInterceptor<*> = protobufInterceptorFactory.create(action)!!
+    val boxResponseInterceptor: Interceptor = boxResponseInterceptorFactory.create(action)!!
```

Comment:
This file's diff seems to be because the formatting changed from 4 to 2 spaces. I _think_ 4 is what we have elsewhere, so we probably want to stay consistent.